setup_chef_cookbooks.sh

```diff
@@ -51,13 +51,13 @@ ENV['no_proxy'] = no_proxy_string
 http_proxy ENV['http_proxy']
 https_proxy ENV['https_proxy']
 no_proxy no_proxy_string
-
+ENV['GIT_SSL_NO_VERIFY'] = 'true'
```

Comment:
This should be covered by [proxy_setup.sh](https://github.com/kiiranh/chef-bach/blob/hannibal/proxy_setup.sh#L21) already...